lib/feed.xml

```diff
@@ -53,28 +53,14 @@
       <content type="html" xml:base="{{ post.url | prepend: url_base | xml_escape }}">{{ post.content | strip | xml_escape }}</content>
 
       {% assign post_author = post.author | default: post.authors[0] | default: site.author %}
-      {% assign post_author_email = false %}
-      {% assign post_author_uri = false %}
+      {% assign post_author = site.data.authors[post_author] | default: post_author %}
+      {% assign post_author_email = post_author.email | default: nil %}
+      {% assign post_author_uri = post_author.uri | default: nil %}
+      {% assign post_author_name = post_author.name | default: post_author %}
```

Comment:
The behavior is the same either way, but I feel like including some sort of `default` is an acknowledgment that this may not be set, and that is ok.

If you do not think it makes the code easier to follow, I can remove it.